app/admin/todo_list.rb

```diff
@@ -33,11 +33,13 @@
   end
 
   form do |f|
-    f.inputs 'Todo List' do
+    f.inputs "Todo List" do
       f.input :title
       f.input :deadline, as: :datepicker, datepicker_options: { min_date: Time.now.to_date }
     end
-
+    f.inputs "Assignees" do
+      f.input :users, :as => :check_boxes, :input_html => { :checked => 'checked' }
```

Comment:
Line is too long. [83/80]<br>Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.